routes/api_events_routes.js

```diff
@@ -101,4 +101,12 @@ module.exports = (app, passport) => {
             log.debug('EventsAPI edit ' + _.get(req, 'params.event'));
         });
 
+    //change the current event id for camps manager or admin 
+    app.post('/events/change', userRole.isCampManager(), (req, res) => {
+        //set the new event id in the session 
+        req.session.passport.user.currentEventId = req.body.currentEventId;
+        req.session.save()
+        res.send(200);
+    });
+
```

Comment:
Why only camp managers or admins can change event id (for himself)?
[I'm thinking about a usecase where a user without privileges wants to view his old camp]

Reply:
that was the request, that only admins and camp managers can change it
at least for now